### DISCUSSION TITLE
How to do social login

### DISCUSSION THREAD

**Discussion by ccrohit** [directus/directus#24159] [Feature Requests]
### Summary

I want to login though the social platform like(Google ,Facebook , git hub like this)

### Motivation

I want to login though the social platform like(Google ,Facebook , git hub like this)

### Detailed Design

I want to login though the social platform like(Google ,Facebook , git hub like this)

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

I want to login though the social platform like(Google ,Facebook , git hub like this)

### Alternatives

I want to login though the social platform like(Google ,Facebook , git hub like this)

### Adoption Strategy

I want to login though the social platform like(Google ,Facebook , git hub like this)

### Unresolved Questions

I want to login though the social platform like(Google ,Facebook , git hub like this)

**Comment by rijkvanzanten** [comment]
See https://docs.directus.io/self-hosted/sso.html
